docker-compose/docker-compose.yml

```diff
@@ -42,44 +42,61 @@ services:
         ports:
             - 8082:8080 #Browser port
 
+    transform-router:
+        image: quay.io/alfresco/alfresco-transform-router:0.0.11-EA
+        environment:
+          ACTIVEMQ_URL: "nio://activemq:61616"
+          IMAGEMAGICK_URL: "http://imagemagick:8090"
+          PDF_RENDERER_URL : "http://alfresco-pdf-renderer:8090"
+          LIBREOFFICE_URL : "http://libreoffice:8090"
+          TIKA_URL : "http://tika:8090"
+        links:
+          - activemq
+
     alfresco-pdf-renderer:
-        image: alfresco/alfresco-pdf-renderer:1.3
+        image: alfresco/alfresco-pdf-renderer:2.0.0-RC2
```

Comment:
Check with @montgolfiere and @pnirwal , but these images should not be public as agreed with management, yesterday.  This compose requires quay.io, anyway.  The original 1.N images can remain in Docker Hub as they are required for the current Enterprise trial.